### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Fix `data replication`, add simple deployment procedure

### What is changed and how it works?

same as above

### Check List 

Tests 

 - No code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,13 @@ Notice ticdc supports building with Go version `Go >= 1.13`
...
 
-## Installing and Deployment
+## Deployment
 
-TODO
+You can setup a CDC cluster for replication test manually as following:
+
+1. setup a TiDB cluster.
+2. start a CDC cluster, which contains one or more CDC servers. The command to start on CDC server is `cdc server --pd-endpoints http://10.0.10.25:2379`, where `http://10.0.10.25:2379` is the client-url of pd-server.
+3. start a replication changefeed by `cdc cli --pd-addr http://10.0.10.25:2379 --start-ts 413105904441098240 --sink-uri root@tcp(127.0.0.1:3306)/test`. The tso is TiDB `timestamp oracle`, if it is not provided, zero will be used. Currently we support MySQL protocol compatible database as downstream sink only, we will add more sink type in the future.
```
Comment (july2993) [inline README.md]: we should use a latest timestamp instead of zero

Comment (amyangfei) [reply README.md]: Does the latest timestamp mean the `timestamp` of now?

Comment (july2993) [reply README.md]: yes,  or fetch a timestamp from pd

Comment (amyangfei) [reply README.md]: fixed, use the current tso

**Pull request merged**